main.go

```
@@ -18,10 +14,7 @@ func main() {
 	r := gin.Default()
 
 	// Health check endpoint
-	r.GET("/", func(c *gin.Context) {
-		response := interfaces.NewHealthResponse()
-		c.JSON(http.StatusOK, response)
-	})
+	r.GET("/", controllers.HealthCheck)
 
 	r.Run(":3000")
```

golangci-lint (1.64.8)

Error return value of `r.Run` is not checked

(errcheck)